src/components/ebay-carousel/style.less

```diff
@@ -141,25 +176,42 @@
     }
 }
 
+@media (hover: none) {
```

Comment:
@DylanPiercey What are we doing for IE11 again? I don't think this media query is supported.

Reply:
This isn’t supported in ie11 and won’t run. It’s fine though because it is meant to detect any device that doesn’t support hover (mobile devices) and ie11 does.

Reply:
Thanks!